src/components/DefinitionEntry.js

```diff
@@ -244,14 +276,7 @@ export default class DefinitionEntry extends React.Component {
           <Row>
             <Col md={2}>{this.renderLabel('Facets', true)}</Col>
             <Col md={10}>
-              <InlineEditor
-                type="text"
-                initialValue={this.printArray(initialFacets)}
-                value={this.printArray(this.getValue('described.facets') || initialFacets)}
-                onChange={this.fieldChange('described.facets', isEqual, this.parseArray)}
-                validator={value => true}
-                placeholder={'Facets'}
-              />
+              <p className="list-singleLine"> &nbsp;&nbsp;&nbsp;&nbsp;{this.printArray(initialFacets)}</p>
```

Comment:
4 &nbsp; ? does that line up well? In the grander scheme of things we should be using CSS etc for layout. OK to be lazy right now as long as it looks reasonable in our default setup.

Reply:
I had to do 4 to make it lineup. The InlineEditor has a pencil icon before the value to represent an editable field. In order to lineup facets with the other values, 4 spaces had to be placed.